### PR TITLE
Updates docs

### PR DESCRIPTION
**Pull request by kodymoodley**
##### Description of changes
Updates existing docs for Actor and Action widgets
Adds docs for Elements widget
Adds docs for installation instructions

##### Includes
- [ ] Code changes
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: orange-story-navigator.code-workspace
Comment (kodymoodley) [inline orange-story-navigator.code-workspace:1]: @ThijsVroegh what is the purpose of this file? Could it be deleted?

File: doc/widgets/install.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+How to enable StoryNavigator in Orange
+=======
+![](../../doc\widgets\images\storynavigator_logo.png)
```
Comment (kodymoodley) [inline doc/widgets/install.md:3]: @ThijsVroegh are you sure the mixing of backslashes and forward slashes in the path is correct? Same goes for the other references to images in the other doc files in this PR.

File: orange-story-navigator.code-workspace
Comment (ThijsVroegh) [reply orange-story-navigator.code-workspace:1]: Yes, this can be deleted

File: doc/widgets/install.md (hunk 1 above)
Comment (ThijsVroegh) [reply doc/widgets/install.md:3]: These are now corrected in all being forward slashes: '/'

**Comment by ThijsVroegh** [comment]
Merging updated documentation

**Pull request merged**